src/scripts/create-parameters.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
```

Comment:
WDYT about making this match the one added by the job that runs it? 
```suggestion
#!/bin/bash -eo pipefail
```

Reply:
I got an error when I tried to do it https://app.circleci.com/pipelines/github/CircleCI-Public/path-filtering-orb/266/workflows/d494de94-71fa-4e6d-bff2-5c1fd9d4d317/jobs/1345
so I used set instead. Does that look good?